src/legacy/core_plugins/kibana/public/doc/components/doc.tsx

```diff
@@ -0,0 +1,87 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React from 'react';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { EuiCallOut, EuiLoadingSpinner, EuiPageContent } from '@elastic/eui';
+import { IndexPattern } from 'src/legacy/core_plugins/data/public/index_patterns/index_patterns';
+import { ElasticSearchHit } from 'ui/registry/doc_views_types';
+import { DocViewer } from '../../doc_viewer/doc_viewer';
+
+export function Doc({
+  status,
+  indexPattern,
+  hit,
+}: {
+  status: string;
+  indexPattern: IndexPattern;
+  hit: ElasticSearchHit;
+}) {
+  return (
+    <EuiPageContent>
+      {status === 'notFound' && (
+        <EuiCallOut
+          title={
+            <FormattedMessage
+              id="kbn.doc.failedToLocateDocumentDescription"
+              defaultMessage="Failed to locate document"
+            />
+          }
+          color="danger"
+          iconType="alert"
+        >
+          <FormattedMessage
+            id="kbn.doc.couldNotFindDocumentsDescription"
+            defaultMessage="Unfortunately I could not find any documents matching that id, of that type, in that index. I tried really hard. I wanted it to be there. Sometimes I swear documents grow legs and just walk out of the index. Sneaky. I wish I could offer some advice here, something to make you feel better"
```

Comment:
@gchaps another message

Reply:
Suggest:

No documents matching that ID and type were found.

Some additional thoughts:
- I’m not sure we need to mention "in the index". I’d assume that’s understood. 
- Types were deprecated in 7.0. So we maybe able to leave off “and type” too.

Reply:
yes, since we deprecate types, no need to mention them. thx for your input!

Reply:
Here we go: 
![image](https://user-images.githubusercontent.com/463851/64864563-b03f6200-d637-11e9-88e2-5762ac60bcca.png)

Reply:
How about using this to remove some of the redundancy 

Cannot find document
No documents match that ID.

Reply:
good choice, done!